### PR TITLE
Easy local development and release process

### PR DESCRIPTION
**Pull request by linuxrider**

### CODE REVIEW

File: rever.xsh [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+$ACTIVITIES = [
+    'version_bump',
+    'changelog',
+    'tag',
+    'push_tag',
+    'ghrelease',
+]
+
+$VERSION_BUMP_PATTERNS = [(file, r'"unreleased$id": "', '"$id": "https://schema-base-url/$VERSION') for sublist in [[os.path.join(i[0], j) for j in i[2]] for i in os.walk('.')] for file in sublist if "json" in file]
```
Comment (saraedum) [inline rever.xsh:45]: That won't work. This rewrites `$id` into the repository with the first release.

**Comment by saraedum** [comment]
Can't we use `--base-uri` for this purpose?

**Comment by linuxrider** [comment]
> Can't we use `--base-uri` for this purpose?

As I understand it, this only has an effect at the top level schema (e.g. `svgdigitizer.json`) but not at the lower levels (e.g. `experimental.json`).

**Comment by linuxrider** [comment]
`--base-uri` is enough as long as we only use our self-contained schemas. As soon as we also use external schemas with  subschemas referenced relatively it becomes a problem.

**Comment by saraedum** [comment]
> `--base-uri` is enough as long as we only use our self-contained schemas. As soon as we also use external schemas with subschemas referenced relatively it becomes a problem.

Sure. Maybe we can discuss ways to fix this once it's a problem.

**Pull request closed without merging**